services/type-utils/src/main/java/datawave/data/normalizer/DiscreteIndexNormalizer.java

```diff
@@ -7,7 +7,7 @@
  *
  * @param <T>
  */
-public interface DiscreteIndexNormalizer<T extends Comparable<T>> extends Normalizer<T> {
+public interface DiscreteIndexNormalizer<T> extends Normalizer<T> {
```

Comment:
1) Because I don't think there's a reason for any normalizer to be comparable (none of our other normalizers implement comparable)
2) It was making it difficult to abstract the core geometry normalizer functionality into an abstract class.